pkg/controllers/machine/link/suite_test.go

```diff
@@ -76,6 +82,10 @@ var _ = BeforeSuite(func() {
 	env = coretest.NewEnvironment(scheme.Scheme, coretest.WithCRDs(apis.CRDs...))
 	unavailableOfferingsCache = awscache.NewUnavailableOfferings()
 	ec2API = &fake.EC2API{}
+	instanceTypeCache = cache.New(awscache.DefaultTTL, awscache.DefaultCleanupInterval)
```

Comment:
Is this cache used anywhere else or can it just be put directly in the `NewProvider` call?